mentat/code_map.py

```diff
@@ -0,0 +1,226 @@
+import json
+import subprocess
+import tempfile
+from pathlib import Path
+from typing import List
+
+from .llm_api import count_tokens
+
+
+def _get_git_files(git_root: str, absolute_paths: bool = False) -> List[str]:
+    git_files_ = (
+        subprocess.check_output(["git", "ls-files"], cwd=git_root)
```

Comment:
If you pass in text=True as an argument, you don't need .decode

Reply:
Ooh nice, didn't know about that!